PurchasesTests/Purchasing/ProductInfoTests.swift

```diff
@@ -150,6 +150,6 @@ class ProductInfoTests: XCTestCase {
                                                introPrice: 0,
                                                subscriptionGroup: "cool_group",
                                                discounts: [discount1, discount2, discount3])
-        expect(productInfo.cacheKey) == "cool_product-49.99-UYU-1-0-cool_group-P3Y-P3W-0-offerid1-offerid2-offerid3"
+        expect(productInfo.cacheKey) == "cool_product-49.99-UYU-1-0-cool_group-P3Y-P3W-2-offerid1-offerid2-offerid3"
```

Comment:
This doesn't need to be stable across versions.

Reply:
Only slightly matters to cached product infos, but no real harm.

Reply:
Are they cached on disk though?